lighthouse-cli/test/fixtures/a11y/a11y_tester.html

```diff
@@ -133,7 +133,7 @@
     <section>
       <div id="duplicate-id-aria" class="duplicate-id-aria"></span>
       <div id="duplicate-id-aria"></div>
-      <div aria-labelledby="duplicate-id-aria"></div>
+      <input type="text" aria-labelledby="duplicate-id-aria"/>
```

Comment:
this needs to be an appropriate use of `aria-labelledby` or this now gets flagged by `aria-allowed-attr`. Nice upgrade for that audit